charts/datasaur/values.yaml

```diff
@@ -160,19 +162,30 @@ jobs:
 
 loki:
   enabled: true
+  alerting: null
+
   auth:
     username: null
     password: null
+
   grafana:
     enabled: false
     grafana.ini:
       server:
-        domain: "{{ .Values.ingress.host }}"
-        root_url: "%(protocol)s://%(domain)s:%(http_port)s/grafana/"
+        root_url: "%(protocol)s://%(domain)s/grafana/"
         serve_from_sub_path: true
+    extraConfigmapMounts:
+      - name: grafana-alert-provisioning
+        mountPath: /etc/grafana/provisioning/alerting
+        configMap: grafana-alert-provisioning
+        subPath: ""
+        readOnly: true
```

Comment:
o ya ini referensinya drmn yak?

Reply:
https://github.com/grafana/helm-charts/blob/main/charts/loki-stack/values.yaml

I scrape from this hell of a documentation @tonoman3g xD